textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/constants/LuceneConstants.java

```diff
@@ -0,0 +1,14 @@
+/**
+ * 
+ */
+package edu.uci.ics.textdb.dataflow.constants;
+
+/**
+ * @author sandeepreddy602
+ *
+ */
+public class LuceneConstants {
+    public static final String INDEX_DIR = "index";

```

Comment:
The Lucene constants will be used in many places. So, I had put it in a different class. If we want to make them configuration values, it will be easy if they are defined at one place.